Second draft

sofer opened the pull request:
@iteles @izaakrogan @minaorangina @des-des @mk4111. 

@rub1e and I have worked up a second draft. Could you all take a look at it and leave your comments here--but don't merge yet.

rub1e committed:
students

@sofer

sofer committed:
cleanup of v2

sofer committed:
Copy changes

rub1e committed:
typos

just a couple of typos

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
+FAC was founded on the principal of reciprocity. That is, the company does not charge fees for its study programmes, but expects its students to contribute to the viability and development of the company in return.
 
-In particular, members are expected to participate in our study programmes and to support our goal of being financially sustainable, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future.
+In particular, members of each cohort of students are expected to take the lead on facilitating the course for the next cohort, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future. Our goal is to be financialy sustainable, and the company cannot survive without the continued support of our graduates.
```
- 'graduates' / 'members of each cohort' are synonymous here.  Perhaps choose one for clarity.
- Not clear what is meant by 'facilitating the course' - does this refer specifically to remaining in the space and teaching, or should this be interpreted more loosely (e.g. contributing to resources)?  If the latter, this should be explicit, not implied.
- Grammar point: "...the company cannot survive without the continued support of **its** graduates"

rub1e replied:
- Agree, should change members to graduates
- I like the mild vagueness of "facilitating" as it could cover everything from teaching to marketing to admin to taking the bins out. AND WASHING UP MUGS
- Disagree with grammar point - both are fine

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
+In particular, members of each cohort of students are expected to take the lead on facilitating the course for the next cohort, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future. Our goal is to be financialy sustainable, and the company cannot survive without the continued support of our graduates.
 
-All members must abide by the following rules of membership.
+In addition to providing a free education, FAC has been successful in finding permanent or freelance work for all of our graduates. Our primary source of income is recruitment fees from companies who hire our graduates, and so graduates are also required to give FAC the opportunity to approach employers on their behalf if we do not already have a relationship with them.
```
- This wording implies a guarantee that employment will be successfully found for graduates in all cases.  It also reads as though the burden of responsibility lies with FAC to find these roles, which was not my understanding - I thought it was down to the individual.  Am I mistaken?  Once again, we should avoid implications and aim for explicitness.
- I feel more context could be given to the FAC Employer Relationship thing, just to explain more clearly why it's so helpful for FAC, versus the cold-calling approach.

rub1e replied:
- agree this is a bit unclear and clunky. It also seems like overkill to jam this point into the intro AND rules AND faq. But there's a reason for that...
- Yeah, maybe introducing the relationship thing isn't needed here - the main point is to hammer home the requirement of not going behind our backs when it comes to applications - so I suppose we could lose the nuance and leave it to the faq

msmichellegar replied:
Is it accurate to say that permanent or freelance work has been successfully found for ALL graduates by Founders and Coders? This implies 16/16 graduates of each cohort have been placed. Given our recruitment/agency strategy is still a work in progress, perhaps we might want to reword this a little more modestly.

Also, genuine question: is recruitment fees _currently_ our primary source of income? Or is this something we're aiming to establish in the future?

rub1e replied:
Yes, fair point - I think it's true to say that at some point we've found some work for every graduate who's **needed/wanted** it.

As far as I remember, recruitment fees are **overwhelmingly** our primary source of income, but I don't have the breakdown

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
+All students must abide by the following rules in exchange for joining FAC:
 
-None of these rules will apply retrospectively.
++ To support new students in their studies (and primarily to help with the full-time study programme); 
```
- Is there a difference between helping on the full-time study programme and supporting new students?  If so, this needs to be outlined explicitly.

rub1e replied:
- Yeah good shout - might be simpler to just say "help out with the fulltime study programme" which would necessarily entail supporting new students

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
-None of these rules will apply retrospectively.
++ To support new students in their studies (and primarily to help with the full-time study programme); 
++ To participate in and support the study programmes that we provide to the general public;
++ To always involve FAC immediately if they are considering applying for a position (or encouraging another student to apply)
```
- I think the tone could be a bit more neutral here.  This currently sounds a bit foreboding.

rub1e replied:
I mean, it doesn't go so far as to say "We know where you live" (which we do)... 

Can you think of wording that makes it clear how important this rule is without crossing that line? I added "immediately" for the sense of urgency, and "always" for the sense of importance - maybe it could live without "always" as that does carry the implication that this is an ongoing requirement when technically it's probably only relevant to their first job

minaorangina replied:
I'd argue a charter is meant to be declarative and instructional, and should remain objective, regardless of the circumstances that surrounds it.  It's obviously definitely true that money is an urgent matter for FAC, but arguably the charter is not the appropriate place for this to be conveyed.  Thoughts?

rub1e replied:
I tend to agree - this is a set of rules and in theory we just need to tell people the rules and answer their questions. But if we did, we'd fall foul of the ongoing debate about tone - I think that adding context makes the slightly bitter pill easier to swallow.

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
++ To support new students in their studies (and primarily to help with the full-time study programme); 
++ To participate in and support the study programmes that we provide to the general public;
++ To always involve FAC immediately if they are considering applying for a position (or encouraging another student to apply)
++ To make all other students and graduates, as well as members of the public, feel welcome in our space.
```
- This should go further - this is a great opportunity to unequivocally outline what it means to make people _'feel welcome the space'_, i.e. condemning _racism, sexism, transphobia, ablism, xenophobia, ageism, religious intolerance_ etc etc.  I think the charter would be incomplete without this - it's very important to give people clear, well-defined guidelines as to what is considered unacceptable behaviour at FAC.

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
 
 **What if I receive an unsolicited offer of work?**    
-Refer them to us and follow our guidelines on how to communicate with prospective employers. [in preparation, but likely to include a recommendation that all members put a note on their linkedin profiles that refers prospective employers to FAC]
+Refer them to us and follow our guidelines on how to communicate with prospective employers. It may be useful to display some expanatory text on your LinkedIn page or other professional platforms - FAC will provide this (but it is not mandatory).
```
- Spelling mistake: **explanatory**
- Am I right in assuming these guidelines have yet to be written?  Would be awesome if instructions on where to find the guidelines were provided (once they exist :smile:)

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
-**This makes me uneasy. I don't want to involve FAC in my conversations with prospective employers.**    
-We understand this. FAC membership may not be the right choice for you.
+**This rule makes me uneasy. I don't want to involve FAC in my conversations with prospective employers.**    
+We understand this, but as our primary source of income, this is a condition of joining the programme.
```
- So far, the charter conveys that the existence of this rule is contingent upon the fact that it's the primary source of income for FAC.  If the income landscape for FAC were to change, I'm wondering what would happen to this rule, if anything?

rub1e replied:
Good point. I think that if we view this document as a condition of acceptance rather than a charter, that issue becomes simpler - we can just change the terms of acceptance as circumstances change.

It would be more of an issue if we wanted this document to be a full and unwavering exposition of our principles

minaorangina commented on README.md:
```diff
@@ -1,67 +1,69 @@
-**How much time do I need to give back each month?**    
-As a minimum, regardless of where you are based, you need to be active in at least one FAC study programme or meetup group. Members are expected to contribute one hour of support to FAC programmes for every day that they are working in the co-working space on their own projects. If you have just completed one of the FACx programmes, then it is your collective responsibility with the rest of your  ...
+**How much time do I need to give back?**    
+As a minimum, regardless of where you are based, you need to be active in at least one FAC study programme or meetup group. If you have just completed one of the FACx programmes, then it is your collective responsibility with the rest of your cohort to facilitate FACx+1.
```
- I believe 'FAC study programme' refers to the 'programmes for the general public' mentioned at the top.  Could use the same terminology, to be clearer.

des-des commented on README.md:
```diff
@@ -1,67 +1,69 @@
 
 **If I am freelancing, do I have to give a proportion of my earnings to FAC?**    
-Not unless the work came through FAC [?], in which case the proportion returned to FAC will be decided on a project-by-project basis.
+Not unless the work came through FAC, in which case the proportion returned to FAC will be decided on a project-by-project basis.
```
I am confused by this. Surely if you have followed the rules set out in the FAQ before this point you will have directed all your freelance job offers through FAC already. Seems a bit contradictory. Could clarify when you would take on freelance work without going through FAC yet still abide by the rules set out here.

rub1e replied:
Interesting. I guess if your mate's slipping you £100 to do a bit of work on her website then FAC shouldn't have much to say on the matter.

On reflection, I think this is a remnant from the original document which had a broader remit than this one. I will tweak this shortly

rub1e committed:
incorporating feedback

Updated with feedback received so far. I think it's important that we comment on this, amend it, and merge it with master asap so people don't end up commenting on outdated versions. Please also note we need to have a proper new draft before Monday 19th Oct as this is when we agreed we will present to the rest of FAC so they have a week to comment on it before it is voted on (or just accepted) at the biz meeting on Monday 26th Oct. (Correct me if I'm wrong on that point)

@msmichellegar @minaorangina I have tried to address the issue of tone, and specifically have changed the bits you pulled out in #20. I have tried to walk the line between being welcoming, but also being declarative, instructional, and objective as Mina suggested in comments on #18.  

Re CoC - I have added this into the rules but I think for now we should revert to the vaguer language of being welcoming as ~~I think~~ I can guarantee there will be much healthy debate before we can agree on a full CoC  

@des-des I have changed that point based on your comment  

@sofer I added something in about London Living Wage which is basically me deciding policy on the fly (but I think it tracks with previous policies) - do take a look.  

-----

For my money, I don't think this should be softened at all. If it were up to me, I'd have it printed, signed, then laminated. I'd turn it into a FACocratic oath that all new students should recite every morning. I think that amid all the avocados and clapping, we need to make it aggressively clear that this is a business that quite literally cannot afford passengers. Nonetheless I have toned it down as much as I can, please do let me know what you think.

sofer committed:
Removed 10% figure

*"The amount is decided on a project-by-project basis, but is usually 10%"* does not fit with the A-Team plans, so have removed.

sofer committed:
Merge pull request #21 from foundersandcoders/v3

incorporating feedback
